src/main/java/games/loveletter/LoveLetterForwardModel.java

```diff
@@ -246,6 +248,9 @@ public boolean checkEndOfGame(LoveLetterGameState llgs) {
             int bestValue = 0;
             for (int i = 0; i < llgs.getNPlayers(); i++) {
                 if (llgs.getPlayerResults()[i] != Utils.GameResult.LOSE) {
+                    if (llgs.playerHandCards.get(i).peek() == null) {
```

Comment:
What's the purpose of this?

Reply:
Ahem...left in from when I was debugging a problem in LoveLetter with MultiTree MCTS. Forgot to remove it...now done.